tests/conftest.py

```diff
@@ -240,6 +247,45 @@ def minimal_attribute_dict():
     }
 
 
+@pytest.fixture()
+def minimal_dataframe_dict():
+    """links transformers to minimal dataframes needed to successfully run transformer.
+    New transformers need to be added here"""
```

Comment:
Q: Do they?  Looks like new transformers in existing modules would be caught automatically by get_all_classes.  Perhaps rephrase to highlight that they need to be added if in a new module or if they do not follow the established pattern in their module?